### PR TITLE
doc: 14.2.22 Release Notes

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/releases/nautilus.rst [hunk 1]
```diff
@@ -5,6 +5,146 @@ Nautilus
...
+* qa/tests: added client-upgrade-nautilus-pacific tests (`pr#39818 <https://github.com/ceph/ceph/pull/39818>`_, Yuri Weinstein)
+* qa/tests: advanced nautilus initial version to 14.2.20 (`pr#41227 <https://github.com/ceph/ceph/pull/41227>`_, Yuri Weinstein)
+* qa/upgrade: disable update_features test_notify with older client as lockowner (`pr#41513 <https://github.com/ceph/ceph/pull/41513>`_, Deepika Upadhyay)
+* qa: add sleep for blocklisting to take effect (`pr#40714 <https://github.com/ceph/ceph/pull/40714>`_, Patrick Donnelly)
+* qa: bump osd heartbeat grace for ffsb workload (`pr#40713 <https://github.com/ceph/ceph/pull/40713>`_, Nathan Cutler)
+* qa: delete all fs during tearDown (`pr#40709 <https://github.com/ceph/ceph/pull/40709>`_, Patrick Donnelly)
+* qa: krbd_blkroset.t: update for separate hw and user read-only flags (`pr#40212 <https://github.com/ceph/ceph/pull/40212>`_, Ilya Dryomov)
+* qa: vstart_runner: TypeError: lstat: path should be string, bytes or os.PathLike, not NoneType (`pr#41485 <https://github.com/ceph/ceph/pull/41485>`_, Patrick Donnelly)
+    * refs/pull/41485/head:
```
Comment (djgalloway) [inline doc/releases/nautilus.rst:123]: Lines like this one should get deleted

Comment (djgalloway) [reply doc/releases/nautilus.rst:123]: Other than that, LGTM provided component leads add their Notable Changes.

**Review by idryomov — COMMENTED** [review]
Need to add an entry to `doc/releases/releases.yml`.

File: doc/releases/nautilus.rst [hunk 2]
```diff
@@ -5,6 +5,144 @@ Nautilus
 Nautilus is the 14th stable release of Ceph.  It is named after the
 nautilus, a family of cephalopods characterized by a whorled shell.
 
+v14.2.22 Nautilus
+=================
+
+
```
Comment (idryomov) [inline doc/releases/nautilus.rst:11]: Should mention that this is likely the last Nautilus release.

Comment (yuriw) [reply doc/releases/nautilus.rst:11]: @idryomov what would I say for `released` date in `doc/releases/releases.yml` ?

Comment (yuriw) [reply doc/releases/nautilus.rst:12]: `Should mention that this is likely the last Nautilus release` done

**Comment by idryomov** [comment]
Fixed a couple of nits, squashed Yuri's commits and added a msgr2 notable change.

File: doc/releases/releases.yml [hunk 3]
```diff
@@ -59,6 +59,8 @@ releases:
   nautilus:
     target_eol: 2021-06-01
     releases:
+      - version: 14.2.22
+        released: 2021-06-30
```
Comment (neha-ojha) [inline doc/releases/releases.yml:63]: might need to adjust this

**Review by neha-ojha — APPROVED** [review]

File: doc/releases/nautilus.rst [hunk 4]
```diff
@@ -5,6 +5,180 @@ Nautilus
 Nautilus is the 14th stable release of Ceph.  It is named after the
 nautilus, a family of cephalopods characterized by a whorled shell.
 
+v14.2.22 Nautilus
+=================
+
+This is the 22nd and likely the last backport release in the Nautilus series.
```
Comment (mykaul) [inline doc/releases/nautilus.rst:11]: 'Users are encouraged to upgrade to newer Ceph versions' ?

**Review by cbodley — APPROVED** [review]

**Review by idryomov — APPROVED** [review]

**Pull request merged**